openedx/core/djangoapps/course_groups/management/commands/remove_users_from_multiple_cohorts.py

```diff
@@ -0,0 +1,56 @@
+"""
+Script for removing users with multiple cohorts of a course from cohorts
+to ensure user's uniqueness for a course cohorts
+"""
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+from django.db.models import Count
+
+from openedx.core.djangoapps.course_groups.models import CourseUserGroup
+
+
+class Command(BaseCommand):
+    """
+    Remove users with multiple cohorts of a course from all cohorts
+    """
+    help = 'Remove all users from multiple cohorts (except one) of each course'
+
+    def handle(self, *args, **options):
+        """
+        Execute the command
+        """
+        # Get entries of cohorts which have same user added multiple times for a single course
+        multiple_objects_cohorts = CourseUserGroup.objects.filter(group_type=CourseUserGroup.COHORT).\
+            values_list('users', 'course_id').annotate(user_count=Count('users')).filter(user_count__gt=1).\
+            order_by('users')
```

Comment:
this query will list each user with each ```CourseUserGroup``` object (of group type ```'cohort'```) with an attribute ```user_count``` (total number of times that user added in that cohort).